qa/SidechainTestFramework/sc_test_framework.py

```diff
@@ -48,25 +49,50 @@ def sync_all(self):
 
     def join_network(self):
         pass
-    
+
     def sc_add_options(self, parser):
         pass
-    
-    #For now it's not active. scutil.generateGenesisData will return None
+
     def sc_generate_genesis_data(self):
```

Comment:
Do we need this anymore? Why not to hide it inside bootstrap related utils.

Reply:
Removed